extras/nest-config.in

```diff
@@ -41,48 +45,43 @@ while test $# -gt 0; do
     --prefix=*)
         prefix=$optarg
         ;;
-
     --prefix)
         echo $prefix
         ;;
-
     --exec-prefix)
         echo $exec_prefix
         ;;
-
     --version)
-        echo @NEST_VERSION_VERSION@
-        exit 0
+        echo "@NEST_VERSION_VERSION@"
         ;;
-
     --help)
         usage 0
         ;;
-
+    --includes)
+        echo "@ALL_INCLUDES@"
+        ;;
     --cflags)
-        NEST_CFLAGS="-I@PKGSRCDIR@/{libnestutil,librandom,sli,nestkernel,nest,topology,precise} @ALL_INCLUDES@"
-        if [ "@PKGBUILDDIR@" != "@PKGSRCDIR@" ]; then
-            NEST_CFLAGS="-I@PKGBUILDDIR@/libnestutil $NEST_CFLAGS"
-        fi
-        NEST_CFLAGS="@ALL_CXXFLAGS@ $NEST_CFLAGS"
-        echo $NEST_CFLAGS
+        echo "@ALL_CXXFLAGS@"
         ;;
-
     --libs)
-        NEST_LDFLAGS="-lnest_lib -lrandom -lsli_lib -lnestkernel -lnestutil -lprecise -ltopology @ALL_LIBS@"
-        # cmake: all libs are installed
-        NEST_LDFLAGS="-L@CMAKE_INSTALL_PREFIX@/@INSTALL_LIB_DIR@ $NEST_LDFLAGS"
-        echo $NEST_LDFLAGS
+        echo "-L@CMAKE_INSTALL_FULL_LIBDIR@ @ALL_LIBS@"
         ;;
-
     --compiler)
-        echo @CMAKE_CXX_COMPILER@
+        echo "@CMAKE_CXX_COMPILER@"
         ;;
 
     --static-libraries)
-        echo @static-libraries@
+        echo "@static-libraries@"
+        ;;
+    --docdir)
+        echo "@CMAKE_INSTALL_DOCDIR@"
```

Comment:
`--docdir` will print the relative path of the location of the documentation, not the absolute path. Is that sensible? If a user wants to know where the documentation is, the absolute path would be required, wouldn't it? The same applies to `--datadir` and `--libdir`.